File: src/main/java/me/lulu/hypixelutilities/craft/QuickCraftExecutor.java
package me.lulu.hypixelutilities.craft;

import me.lulu.hypixelutilities.util.McUtil;
import net.minecraft.client.entity.EntityPlayerSP;

public class QuickCraftExecutor {

    public static void execute(QuickCraft craft) {
        EntityPlayerSP player = McUtil.getPlayer();
        player.sendChatMessage("/internal_autocraftitem " + craft.getNameInHypixel());
    }

}

File: src/main/java/me/lulu/hypixelutilities/craft/QuickCraftExecutor.kt
package me.lulu.hypixelutilities.craft

import me.lulu.hypixelutilities.util.clientPlayer

object QuickCraftExecutor {

    fun execute(craft: QuickCraft) {
        clientPlayer.sendChatMessage("/internal_autocraftitem " + craft.nameInHypixel)
    }

}
